avocado_vt/test.py

```diff
@@ -158,8 +158,8 @@ def runTest(self):
             self._runTest()
         # This trick will give better reporting of virt tests being executed
         # into avocado (skips, warns and errors will display correctly)
-        except error.TestNAError, details:
-            raise exceptions.TestNAError(details)
+        except error.TestSkipError, details:
+            raise exceptions.TestSkipError(details)
```

Comment:
@apahim, actually @will-Do is correct. The first line should continue to be

     except error.TestNAError, details:

Instead of what you have changed it to. The reason is that this exception comes from autotest, not avocado.